config_complete.toml

```diff
@@ -143,6 +143,16 @@ AdditionalErrorFields = [
 # Eligible for live reload.
 # AddSpanCountToRoot = true
 
+# UseStableCacheManagement controls the use of a non-resizing cache management strategy.
+# It defaults to false, but is recommended to set to true, as the old strategy does not recover from
+# cache overruns.
+# In the previous strategy, under memory pressure the cache is shrunk and never grows again.
+# In the stable strategy, under memory pressure the items having the most impact on the cache size are
+# ejected from the cache early and the cache is not resized.
+# This only applies if MaxAlloc is nonzero.
+# Eligible for live reload.
+# UseStableCacheManagement = true
```

Comment:
I feel like having a boolean here isn't too future proof - if we have to explore other strategies in the future, it means deprecating this option. An alternative config option would be something like "CacheEvictionStrategy" with the default being the existing "Resizing" and this one being "CacheImpact" (names are just suggestions).

Reply:
That's a really good point, and I will implement it. Thank you.